fcd/ast/function.h

```diff
@@ -60,14 +60,14 @@ class FunctionNode
 	typedef decltype(declarations)::iterator declaration_iterator;
 	
 	DumbAllocator pool;
-	Statement* body;
+	Statement* body = nullptr;
 	
 	static void printIntegerConstant(llvm::raw_ostream&& os, uint64_t constant);
 	static void printIntegerConstant(llvm::raw_ostream& os, uint64_t constant);
 	static void printPrototype(llvm::raw_ostream& os, llvm::Function& function, llvm::Type* returnType = nullptr);
 	
 	inline FunctionNode(llvm::Function& fn)
-	: function(fn), body(nullptr), returnType(nullptr)
+	: function(fn)
```

Comment:
Yeah well the problem is code changes over time.
Constructors/members get added and it's easy to forget initializations.
Also the only member that really gets assigned is function. The rest just makes it harder to focus on what the constructor does.